unesp-dracena.csl

```diff
@@ -0,0 +1,589 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
+  <info>
+    <title>UNESP - Campus de Dracena - ABNT - Autores abreviados (Portuguese - Brazil)</title>
+    <title-short>UNESP-FCAT</title-short>
+    <id>http://www.zotero.org/styles/unesp-dracena</id>
+    <link href="http://www.zotero.org/styles/unesp-dracena" rel="self"/>
+    <link href="http://www.zotero.org/styles/unesp-dracena.csl" rel="template"/>
```

Comment:
the test is failing as this is not a template style. Either include the style you used as a base for this here style or remove line 8. (better to include the link to the template style as this gives us a history and also honours the work of the maker of the template style.)
Also, no need to submit this 4 times, you can just edit the one style here and close the other submissions.